### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by jdecool**
The CI is red and this PR fix it.

There's 3 problems:
* Thruway docker image (`docker/thruway/Dockerfile`) use `voryx/thruway` which also uses `thruway/client` which is not currently compatible with PHP 8.0. So before the library is compatible, I've fixed the PHP version to 7.4 for this container.
* `docker/Dockerfile` image is executed with PHP 7.4 & 8.0 and use `PEAR` to install some extensions. As `PR` is not fully compatible with PHP 8.0, required dependencies are compiled.
* `enqueue` libraries depends on `thruway/pawl-transport` which depend on `ratchet/pawl` (`^0.3.1`). The 0.3.x version doesn't support `guzzlehttp/psr7` 2.x branch. So we needs to restrict `guzzlehttp/psr7` to `1.x` branch

### CODE REVIEW

**Comment by jdecool** [comment]
Strange. I've tested it on my fork before and it was green https://github.com/jdecool/enqueue-dev/pull/1/checks

**Comment by jdecool** [comment]
Could you retry the failed job ?

File: docker/Dockerfile [hunk 1]
```diff
@@ -37,16 +46,19 @@ RUN set -x && \
...
-        g++ \
-    && \
     mkdir -p $HOME/librdkafka && \
     cd $HOME/librdkafka && \
     git clone https://github.com/edenhill/librdkafka.git . && \
     git checkout v1.0.0 && \
-    ./configure && make && make install && \
-    pecl install rdkafka && \
+    ./configure && make && make install
+
```
Comment (Steveb-p) [inline docker/Dockerfile:54]: Using `pecl install rdkafka-5.0.2` was not working?

**Review by Steveb-p — APPROVED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (jdecool) [reply docker/Dockerfile:54]: With PHP 8.x when using `pecl` it produce a PHP warning (`Invalid argument supplied for foreach() in /usr/share/pear/PEAR/...`) that break the image build.

That's why I move it to compile it manually.

**Comment by makasim** [comment]
Thanks!

**Pull request merged**